core/client/fs/src/main/java/alluxio/client/block/policy/BlockLocationPolicy.java

```diff
@@ -66,7 +67,8 @@ public static BlockLocationPolicy create(CreateOptions options) {
    * Gets the worker's network address for serving operations requested for the block.
    *
    * @param options the options to get a block worker network address for a block
-   * @return the address of the worker to write to, null if no worker can be selected
+   * @return the address of the worker to write to
+   * @throws UnavailableException if there are no workers eligible to serve the request
    */
-  WorkerNetAddress getWorker(GetWorkerOptions options);
+  WorkerNetAddress getWorker(GetWorkerOptions options) throws UnavailableException;
```

Comment:
We are not using this annotation consistently in the code base at the moment, I would prefer to update these when we set the guideline.